association_backend/association/domain/exceptions.py

```diff
@@ -0,0 +1,37 @@
+import pydantic
+
+
+class StripeCheckoutSessionNotCreated(Exception):
```

Comment:
would be good to get some documentation that explains when those exceptions are raised and why 🤔

Reply:
Ok, see [a8f1e453](https://github.com/madEng84/pycon/commit/a8f1e453cb85469456816d449337074e37b9ecf4)